notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -4014,6 +4014,46 @@ nuclear weapons [A newspaper headline from the future stated that the United Sta
 intellectual property in society [Jennings consented that all his work done at the Nexim corporation was the intellectual property of the corporation under the terms of his non-disclosure agreement.]
 
 
+movie: After the Apocalypse (2004)
+==================================
+
+:: Title
+After the Apocalypse
+
+:: Description
+After the Apocalypse is a 2004 science fiction black-and-white film written
+and directed by Yasuaki Nakajima. The film does not have any dialogue.
+
+Synopsis: Five survivors of an apocalyptic war, a single woman and four men,
+re forced to communicate without words because of destructive gasses from the
+war.
+
+:: Date
+2004
+
+:: References
+https://en.wikipedia.org/wiki/After_the_Apocalypse
+
+:: Ratings
+3 <paul>
+
+:: Major Themes
+annihilation by WMDs [The story is set in the immediate aftermath of an apocalyptic war.]
+surviving in a wasteland [Five survivors of an apocalyptic war tried to make a go of things in the ruins of a destroyed semi-urban area.]
+rivalry [The Asian man and the injured white man were competing for the woman's affections.]
```

Comment:
Sounds like the "male-male" sub-variety of "rivalry"..?

Reply:
Good call. I added **love triangle** to boot.